CONTRIBUTING.md

```diff
@@ -489,6 +490,35 @@ Your PR is now updated on GitHub. To ensure team member are aware of this,
 leave a message on the PR stating something like, "review feedback applied".
 Then, the team is notified and able to re-review your PR more quickly.
 
+### Temporarily blocking a PR
+
+Sometimes a submitter or reviewer of a PR may need to temporarily block
+a PR from being merged. There can be a number of reasons for this, including:
+
+- The PR has been opened as a Request For Comments (RFC).
+- The PR is incomplete, needs more work or rework.
+- The PR is a Work In Progress (WIP), which has been opened
+  to enable others to see progress and provide early feedback.
+
+Kata Containers CI systems have two methods that allow marking
+PRs as such. The methods are as either
+[GibHub labels](https://help.github.com/articles/about-labels/)
+or keywords in the PR subject line. The keywords can appear anywhere
+in the subject line.
+
+The following table summarises the valid items for Kata Containers PRs:
+
+| Type | Value |
+| ---- | ----- |
+| Label | `do-not-merge` |
```

Comment:
Might be worth mentioning somewhere that  we often refer to the `do-not-merge` label as "dnm" on github issues.